diff --git a/node_modules/xstate/dist/declarations/src/State.d.ts b/node_modules/xstate/dist/declarations/src/State.d.ts
index 4b77038..42278b0 100644
--- a/node_modules/xstate/dist/declarations/src/State.d.ts
+++ b/node_modules/xstate/dist/declarations/src/State.d.ts
@@ -66,7 +66,7 @@ interface MachineSnapshotBase<TContext extends MachineContext, TEvent extends Ev
      * @param event The event to test
      * @returns Whether the event will cause a transition
      */
-    can: (event: TEvent) => boolean;
+    can: (event: Pick<TEvent, 'type'>) => boolean;
     getMeta: () => Record<string, any>;
     toJSON: () => unknown;
 }
